front/src/reducers/osrdconf/stdcmConf/index.ts

```diff
@@ -232,10 +246,15 @@ export const stdcmConfSlice = createSlice({
       );
       state.stdcmPathSteps = newPathSteps;
     },
-    // temporary solution to add a new simulation, only 1 action should remain after the refactoring
-    addNewStdcmResult(state: Draft<OsrdStdcmConfState>, action: PayloadAction<StdcmSimulation>) {
-      state.simulations.push(action.payload);
-      state.selectedSimulationIndex = state.simulations.length - 1;
+    createStdcmSimulation(
```

Comment:
Maybe it's more relevant ?
```suggestion
    addStdcmSimulation(
```